CfpExchange/Middleware/RssMiddleware.cs

```diff
@@ -54,14 +54,12 @@ public async Task Invoke(HttpContext context)
 				rssContent += "</channel>" +
 					"</rss>";
 
-				using (var memoryStream = new MemoryStream())
```

Comment:
Don't you want to export this to a method? You used that twice already. Or do you expect that these are the only two occurrences that will be in the solution?